docs/data/data-core-editor.md

```diff
@@ -1,6 +1,6 @@
 # **core/editor**: The Editor’s Data
 
-## Selectors
+## Selectors 
```

Comment:
Trailing whitespace! 😱

Reply:
This is generated, we might have an issue in the build tool